docs/GraFx-Fonts/guides/upload-fonts/index.md

```diff
@@ -0,0 +1,71 @@
+# Upload fonts
+
+## License
+
+Adding fonts to CHILI GraFx requires a valid license. Before uploading, check your font supplies (foundry) if you have a valid license to use your font's for your application.
+
+## Adding font to GraFx Fonts
+
+Click the upload button
+
+![appscreen](uploadbutton.png)
+
+Select 1 or more files to upload
+
+![appscreen](selectfiles.png)
+
+Confirm you have the license.
+
+![appscreen](confirmlicense.png)
+
+Listen carefully while uploading.
+
+![appscreen](elevator.png)
+
+GraFx Fonts will read the metadata from the font files, als suggest the categorization of the fons according the family and [style](https://en.wikipedia.org/wiki/Font#Characteristics).
+
+![appscreen](confirmstyles.png)
+
+At this moment, you can still delete a specific font before it's added to the GraFx Fonts repository.
```

Comment:
please clarify, does this mean it won't be possible in the future ? (just playing dumb client here)

Reply:
Referring "at this moment in the upload process". But good catch.